addons/addon-base-raas/packages/base-raas-cfn-templates/src/templates/onboard-account.cfn.yml

```diff
@@ -801,6 +827,10 @@ Resources:
     Properties:
       SubnetIds:
         - !Ref PrivateWorkspaceSubnet
+        - !If
+          - isAppStreamAndCustomDomain
+          - !Ref PrivateWorkspaceSubnet2
```

Comment:
Having a little difficulty in following why we need this. What is the point of adding this subnet to these interface endpoints?

Reply:
Removing.